apps/evm/lib/evm/operation/system.ex

```diff
@@ -84,15 +84,18 @@ defmodule EVM.Operation.System do
   def call([call_gas, to, value, in_offset, in_size, out_offset, out_size], %{
         exec_env: exec_env,
         machine_state: machine_state
-      }) do
+           }) do
     to = Address.new(to)
     {data, machine_state} = EVM.Memory.read(machine_state, in_offset, in_size)
 
     call_gas = if value != 0, do: call_gas + Gas.callstipend(), else: call_gas
 
+    words = Memory.get_active_words(out_offset + out_size)
+    updated_machine_state = MachineState.maybe_set_active_words(machine_state, words)
```

Comment:
Should we push this logic into `EVM.MessageCall` or is this specific to `call` and doesn't apply to `delegate_call` and `static_call`?